docs/features/exporttopdf/README.md

```diff
@@ -0,0 +1,145 @@
+---
+category: working-with-document
+order: 25
+url: features/exporttopdf
+menu-title: Export to PDF
+meta-title-short: Export to PDF
+---
+<!--
+Copyright (c) 2003-2020, CKSource - Frederico Knabben. All rights reserved.
+For licensing, see LICENSE.md.
+-->
+
+# Exporting content to PDF
+
+<info-box info="">
+	This feature is provided through the [Export to PDF](https://ckeditor.com/cke4/addon/exportpdf) plugin that is not included in any official CKEditor 4 preset. You can {@link guide/dev/plugins/README add it to your custom build} with [online builder](https://ckeditor.com/cke4/builder) or download as a [npm package](https://www.npmjs.com/package/ckeditor4-plugin-exportpdf).
+</info-box>
+
+The [Export to PDF](https://ckeditor.com/cke4/addon/exportpdf) plugin allows you to export the content from editor to a PDF file.
+
+{@img assets/img/exportpdf_01.png Export to PDF button.}
+
+When enabled, it adds the **<img class="inline" src="%BASE_PATH%/assets/img/exportpdf-button.png" alt="Export to PDF toolbar button">Export to PDF** toolbar button. It sends data from editor to the PDF service maintained by [CKEditor Cloud Services](https://ckeditor.com/ckeditor-cloud-services/) and handles the response, saving it as a PDF file.
+
+## Feature overview
+
+CKEditor 4 is a feature rich WYSIWYG editor where you operate mostly on rich text and HTML markup. In many cases pure HTML output is enough, however you may also need to preserve the document in other formats which can be easily shared and presented in different environments. Thanks to the [Export to PDF](https://ckeditor.com/cke4/addon/exportpdf) plugin you can save the document you created to a downloadable file, preserving all the styles and structure in nearly 1:1 projection.
+
+## Plugin setup
+
+### Installation with npm
+
+The Export to PDF plugin for CKEditor 4 is available through [npm](https://www.npmjs.com/package/ckeditor4-plugin-exportpdf). To use it, install the `ckeditor4-plugin-exportpdf` npm package as a dependency of your project:
+
+```plaintext
+npm install ckeditor4-plugin-exportpdf
+```
+
+If CKEditor 4 was also installed through npm (using the `ckeditor4` package) in your project, the plugin will create a symbolic link itself in the CKEditor 4 `plugins/` directory so you do not need to move or copy any files.
+
+If you chose another method to install CKEditor 4, copy or link the `exportpdf` folder from the `ckeditor4-plugin-exportpdf` package to the CKEditor 4 `plugins/` directory.
+
+If you prefer not to touch any files, you may use the [`CKEDITOR.plugins.addExternal()`](https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_plugins.html#method-addExternal) API instead to load the Export to PDF plugin directly from the `node_modules/` directory:
+
+```js
+CKEDITOR.plugins.addExternal( 'exportpdf', './node-modules/ckeditor4-plugin-exportpdf/exportpdf/' );
```

Comment:
```suggestion
CKEDITOR.plugins.addExternal( 'exportpdf', './node-modules/ckeditor4-plugin-exportpdf/' );
```

This path is shorter :point\_up: In Spreadsheets plugin there was specific files structure but in this case we will have regular one so `plugin.js` file is in the root plugin/package directory.

Reply:
I wasn't sure about it, thx :+1: